templates/_asset.html

```diff
@@ -12,23 +12,24 @@
     <p><strong>{{ asset.file_path }}</strong></p>
     <p>
       <span>
-      <a href="/manager/update?file-path={{ asset.file_path }}">Tags</a>:
-      {% for tag in asset.tags %}
+        <a href="/manager/update?file-path={{ asset.file_path }}">Tags</a>:
+        {% for tag in asset.tags %}
         <a href="/manager?tag={{ tag.name }}">
           <span class="p-label">{{tag.name}}</span>
         </a>
-      {% endfor %}
+        {% endfor %}
       </span>
     </p>
     <p><span>Created: {{ asset.created }}</span></p>
     <p>
       <small>Copy:</small>
       <button
         class="p-button--positive copy-link is-inline"
-        data-url="/{{ asset.file_path }}"
+        data-url="/v1/{{ asset.file_path }}"
       >
         Link
       </button>
+      <!-- TODO: I'm not sure about this button's expected behaviour -->
```

Comment:
Trying to run it now, I'll see if I can get to the bottom of this for you.

Reply:
Ah I have a feeling that this is supposed to generate a template syntax compatible with the [image-template](https://github.com/canonical/canonicalwebteam.image-template) module, so you can paste it into a Jinja template with all the right screensize optimisations etc.

But I'd be cool with simply commenting this out for now, and filing an issue to re-enable it as a follow-up.